modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +27,15 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self):
```

Comment:
Can you add the argument include_optional=False?
Then if include_optional is passed in, please include the optional properties too.
If it is omitted, please include only the required properties.

Reply:
Can you show an example on how to do that in the best way?
Anyway, I will have it set to true by default, since it is necessary to support default tests that include Discriminators (AnyOf, etc).

Reply:
Ok, I saw what you meant in the following comment. It's not adding a flag to the generator and then pushing the flag in the models, but instead just changing the signature of the functions. Done

Reply:
Can you resolve this conversation?